manifests/candlepin.pp

```diff
@@ -105,8 +105,7 @@
       target => $keystore,
       owner  => 'tomcat',
       group  => $group,
-    }
-
+    } ->
```

Comment:
One the one hand yes, on the other the current solution does not hurt and given the current structure of the file, one can be easily tricked into thinking that everything is executed in order (which was not true before my commit).